Percy/Percy.cs

```diff
@@ -139,11 +148,12 @@ private static string GetPercyDOM()
 
         public class Options : Dictionary<string, object> {}
 
-        public static void Snapshot(
+        public static JObject Snapshot(
             WebDriver driver, string name,
-            IEnumerable<KeyValuePair<string, object>>? options = null)
+            IEnumerable<KeyValuePair<string, object>>? options = null,
+            Boolean sync = false)
```

Comment:
sync should be inside options as other parameter

Reply:
done